runware/types.py

```diff
@@ -142,147 +142,68 @@ class IRefiner:
     startStepPercentage: Optional[float] = None
 
 
-@dataclass
+@dataclass(kw_only=True)
 class IControlNetGeneral:
-    weight: Optional[float]
-    start_step: Optional[int]
-    end_step: Optional[int]
-    control_mode: Optional[EControlMode]
-    preprocessor: EPreProcessor
-    guide_image: Optional[Union[str, File]] = None
-    guide_image_unprocessed: Optional[Union[str, File]] = None
+    model: Optional[str] = None
+    weight: Optional[float] = None
+    startStep: Optional[int] = None
+    endStep: Optional[int] = None
+    startStepPercentage: Optional[int] = None
+    endStepPercentage: Optional[int] = None
+    controlMode: Optional[EControlMode] = None
+    guideImage: Optional[Union[str, File]] = None
+    guideImageUnprocessed: Optional[Union[str, File]] = None
```

Comment:
please remove this, is not needed.

guideImage can hold both processed and unprocessed images, and we control if preprocess is needed based on the presence of `preprocessor` parameter. Let's try to avoid adding surface/parameters when is not needed please!